error extracting information from jira

I want to extract the title of the jira ticket.

I have tried to do it through jenkins but I am not getting an error message and the fields appear without loading jira info:

- name: Jira
-  title:
- description:


I have also tried using npx
`npx git-changelog-command-line -std  --show-debug-info --jiraServer ...  -jba ...  -fc bc03c141b4746ca  -tc bc03c141b4746ca -jp "MYTICKET_CS-([0-9]+)\\b"  ...`

But throw the following error:
```
[main] INFO shadow.se.bjurr.gitchangelog.internal.integrations.rest.RestClient - GET:
https://..../MYTICKET_CS-72195?fields=parent,summary,issuetype,labels,description,issuelinks
[main] ERROR shadow.se.bjurr.gitchangelog.internal.issues.IssueParser - MYTICKET_CS-72195
shadow.se.bjurr.gitchangelog.api.exceptions.GitChangelogIntegrationException: Unable to parse:

	at shadow.se.bjurr.gitchangelog.internal.integrations.jira.DefaultJiraClient.getIssue(DefaultJiraClient.java:45)
	at shadow.se.bjurr.gitchangelog.internal.issues.IssueParser.createParsedIssue(IssueParser.java:251)
	at shadow.se.bjurr.gitchangelog.internal.issues.IssueParser.parseForIssues(IssueParser.java:88)
	at shadow.se.bjurr.gitchangelog.api.GitChangelogApi.getChangelog(GitChangelogApi.java:628)
	at shadow.se.bjurr.gitchangelog.api.GitChangelogApi.getChangelog(GitChangelogApi.java:83)
	at shadow.se.bjurr.gitchangelog.api.GitChangelogApi.render(GitChangelogApi.java:109)
	at shadow.se.bjurr.gitchangelog.api.GitChangelogApi.render(GitChangelogApi.java:140)
	at se.bjurr.gitchangelog.main.Main.main(Main.java:611)
Caused by: java.lang.IllegalArgumentException: json string can not be null or empty
	at com.jayway.jsonpath.internal.Utils.notEmpty(Utils.java:386)
	at com.jayway.jsonpath.internal.JsonContext.parse(JsonContext.java:81)
	at com.jayway.jsonpath.JsonPath.read(JsonPath.java:502)
	at shadow.se.bjurr.gitchangelog.internal.integrations.jira.JiraClient.toJiraIssue(JiraClient.java:38)
	at shadow.se.bjurr.gitchangelog.internal.integrations.jira.DefaultJiraClient.getIssue(DefaultJiraClient.java:42)
	... 7 more
```

but if I do a get to  https://..../MYTICKET_CS-72195?fields=parent,summary,issuetype,labels,description,issuelinks I get a well formed json

--- a/src/main/java/se/bjurr/gitchangelog/internal/integrations/rest/RestClient.java
+++ b/src/main/java/se/bjurr/gitchangelog/internal/integrations/rest/RestClient.java
@@ -2,7 +2,9 @@ package se.bjurr.gitchangelog.internal.integrations.rest;
 
 import static org.slf4j.LoggerFactory.getLogger;
 
+import java.io.BufferedReader;
 import java.io.InputStream;
+import java.io.InputStreamReader;
 import java.io.UnsupportedEncodingException;
 import java.net.HttpURLConnection;
 import java.net.URL;
@@ -102,9 +104,9 @@ public class RestClient {
   protected String getResponse(final HttpURLConnection conn) throws Exception {
     if (mockedRestClient == null) {
       final InputStream inputStream = conn.getInputStream();
-      final byte[] targetArray = new byte[inputStream.available()];
-      inputStream.read(targetArray);
-      return new String(targetArray, StandardCharsets.UTF_8);
+      InputStreamReader inputStreamReader = new InputStreamReader(inputStream, StandardCharsets.UTF_8);
+      BufferedReader bufferedReader = new BufferedReader(inputStreamReader);
+      return bufferedReader.readLine();
     }
     return mockedRestClient.getResponse(conn);
   }

